### PR TITLE
When you go to the submit pathway and the episode has already been su…

### PR DESCRIPTION
**Pull request by fredkingham**
…ccessfully submitted, show this in the UI

### CODE REVIEW

**Comment by fredkingham** [comment]
Before
Screen Shot 2020-02-27 at 16 31 23

After
Screen Shot 2020-02-27 at 16 31 12

(And the button is removed if it has been submitted)

File: odonto/pathways.py [hunk 1]
```diff
@@ -5,11 +5,22 @@
 from django.db import transaction
 from opal.core import menus, pathway
 from odonto import models
+from odonto.odonto_submissions import models as submission_models
 from odonto.episode_categories import FP17Episode, FP17OEpisode
 from odonto.odonto_submissions import serializers
 from plugins.add_patient_step import FindPatientStep
 
 
+def get_submission_state(episode):
```
Comment (davidmiller) [inline odonto/pathways.py]: This function doesn't get the submission state though... it's `is_submitted` ?

Comment (fredkingham) [reply odonto/pathways.py]: 👍

File: odonto/pathways.py [hunk 2]
```diff
@@ -319,10 +335,15 @@ def to_dict(self, *args, **kwargs):
...
-        check_steps_dict = next(
-            i for i in to_dicted["steps"] if i["step_controller"] == CHECK_STEP_FP17_O.get_step_controller()
-        )
-        check_steps_dict["overlapping_dates"] = self.get_overlapping_dates(patient, episode)
+        check_index = None
+        for index, step_dict in enumerate(to_dicted["steps"]):
+            if step_dict["step_controller"] == CHECK_STEP_FP17_O.get_step_controller():
+                check_index = index
+
+        to_dicted["steps"][check_index]
```
Comment (davidmiller) [inline odonto/pathways.py:343]: This line can go?

**Pull request merged**